content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -75,8 +75,12 @@ The following sysctls are supported in the _safe_ set:
 - `net.ipv4.ip_local_port_range`,
 - `net.ipv4.tcp_syncookies`,
 - `net.ipv4.ping_group_range` (since Kubernetes 1.18),
-- `net.ipv4.ip_unprivileged_port_start` (since Kubernetes 1.22).
-- `net.ipv4.ip_local_reserved_ports` (since Kubernetes 1.27).
+- `net.ipv4.ip_unprivileged_port_start` (since Kubernetes 1.22),
+- `net.ipv4.ip_local_reserved_ports` (since Kubernetes 1.27, needs kernel 3.16+),
+- `net.ipv4.tcp_keepalive_time` (since Kubernetes 1.29, needs kernel 4.5+),
+- `net.ipv4.tcp_fin_timeout` (since Kubernetes 1.29, needs kernel 4.6+),
+- `net.ipv4.tcp_keepalive_intvl` (since Kubernetes 1.29, needs kernel 4.5+),
+- `net.ipv4.tcp_keepalive_probes` (since Kubernetes 1.29, needs kernel 4.5+).
```

Comment:
nit: I'd use a semicolon (`;`) not a comma (`,`)

Also, Linux 4.14 is the oldest LTS kernel. I wonder if we can somehow account for that - maybe a **future** PR can help readers spot what is / isn't supported upstream.

Reply:
Semicolon and comma formats are mixed in this repo. Do you think it should be modified here ?

Reply:
I do, but only as a nit (less significant feedback)

Reply:
done